doc/api-doc.yml

```diff
@@ -26,15 +25,15 @@ paths:
     get:
       description: Returns metrics of the Mira service
       produces:
-        - application/json; charset=utf-8
+        - application/json
```

Comment:
Changed to only application/json since that is what our metrics library gives out.

Reply:
Also JSON according to the spec must be utf-8 so it should be implicit